packages/components/src/modal/Modal.js

```diff
@@ -1,34 +1,54 @@
 // @flow
 
 import * as React from 'react';
-import { View } from 'react-native';
-import { StyleSheet } from '@kiwicom/universal-components';
+import { View, Dimensions, Platform } from 'react-native';
+import { StyleSheet, Modal as UCModal } from '@kiwicom/universal-components';
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 
-import type { ModalProps } from './ModalTypes';
-import ModalWrap from './ModalWrap';
+export type Props = {|
+  +children: React.Node,
+  +isVisible: boolean,
+  +onClose?: () => void,
+|};
 
-const Modal = (props: ModalProps) => {
+export default function Modal({ children, isVisible, onClose }: Props) {
+  const { width, height } = Dimensions.get('window');
+  // Center modal content on web and tablets
+  const centerContent = Platform.OS === 'web' || height / width < 1.6;
```

Comment:
Should we maybe put this logic into some module, so we can say `Device.isTablet()`?

Reply:
I agree.

Reply:
Good idea. Extracted to `Device` inside `margarita-utils`.

I'm also considering something similar for quite frequently repeating `Platform.OS === 'web'`.
What about our own `Platform` implementation with something like `Platform.isWeb()` (isMobile, isAndroid, isIos)?

Reply:
Sure, why not

Reply:
👍 ok, I'll create separate PR for it.